src/index.d.ts

```diff
@@ -108,7 +108,7 @@ declare namespace SDKStandardComponents {
         dfspId: string;
         jwsSign: boolean;
         jwsSignPutParties?: boolean;
-        jwsSigningKey?: string
+        jwsSigningKey?: Buffer
```

Comment:
`env.get('JWS_SIGNING_KEY_PATH').asFileContent()` from `sdk-scheme-adapter`

So I believe this should be a `Buffer`